src/main/java/com/example/book_rest_api/api/service/BookService.java

```diff
@@ -0,0 +1,47 @@
+package com.example.book_rest_api.api.service;
+
+
+import com.example.book_rest_api.api.model.Book;
+import com.example.book_rest_api.api.repository.BookRepository;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import java.util.Optional;
+
+@Service
+public class BookService {
```

Comment:
In a service you shouldn't work with response entity model. Only use your model (Book)